native/YDotNet.Native.MacOS/YDotNet.Native.MacOS.csproj

```diff
@@ -7,7 +7,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Content Include="..\..\output\macos\libyrs.dylib" Pack="True" PackagePath="runtimes\osx\native\%(Filename)%(Extension)" />
+    <Content Include="..\..\output\macos\libyrs.dylib" Pack="True" PackagePath="runtimes\osx-x64\native\%(Filename)%(Extension)" />
+    <Content Include=".\macos-arm64\libyrs.dylib" Pack="True" PackagePath="runtimes\osx-arm64\native\%(Filename)%(Extension)" />
```

Comment:
This should work, but I think we should make a separate package, so you can decide what you need and you have the chance to keep your binary small.